### PR TITLE
Dockerize

### PR DESCRIPTION
**Pull request by HRNN99**
dockerize front & back + envioronment var change

### CODE REVIEW

File: postgre-docker/.dockerignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/front-end/node_modules
```
Comment (wet333) [inline postgre-docker/.dockerignore:1]: what is this hernan? @HRNN99

**Review by wet333 — CHANGES_REQUESTED** [review]
plase read the comments

File: postgre-docker/.dockerignore (hunk 1 above)
Comment (HRNN99) [reply postgre-docker/.dockerignore:1]: Exludes this folder  :)

Comment (wet333) [reply postgre-docker/.dockerignore:1]: oka

**Review by wet333 — APPROVED** [review]

**Comment by wet333** [comment]
created docker-compose for the entire aplication

**Pull request merged**